app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  belongs_to :user
```

Comment:
### Optional
- [ ] It would be better if we call the update method every time a new post is created and saved to the database and we can achieve that with the Rails Active Callbacks `after_save` method. So it would be like `after_save: update_post_counter`

Reply:
Done 👍